internal/quaiapi/quai_api.go

```diff
@@ -125,9 +125,6 @@ func (s *PublicBlockChainQuaiAPI) BlockNumber() hexutil.Uint64 {
 // given block number. The rpc.LatestBlockNumber and rpc.PendingBlockNumber meta
 // block numbers are also allowed.
 func (s *PublicBlockChainQuaiAPI) GetBalance(ctx context.Context, address common.MixedcaseAddress, blockNrOrHash rpc.BlockNumberOrHash) (*hexutil.Big, error) {
-	if !address.ValidChecksum() {
```

Comment:
Why remove? If an address is failing the check we should figure out why

Reply:
good catch, my address was not working in this call, i will revert this change